pinc/gettext_setup.inc

```diff
@@ -33,7 +33,7 @@ function get_desired_language()
     if (isset($_GET['lang']) && lang_code($_GET['lang'])) {
         // User requested a specific language for this page
         $locale = lang_code($_GET['lang']);
-        setcookie("language", $locale, time()+31536000, "/", "", $use_secure_cookies);
+        dp_setcookie("language", $locale, time()+31536000);
```

Comment:
Don't forget to remove the `global $use_secure_cookies` on line 23.

Reply:
This is what happens when you rush a PR while your child takes a nap :).